README.md

```diff
@@ -1,14 +1,25 @@
 # OpenTelemetry Lambda
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Java)?label=CI%20%28Java%29&style=for-the-badge)
```

Comment:
Would it be possible for you to also add one for the Python PR build? 🙂

Also do these badges check the `main` branch by default?

Reply:
Yes I can and I'm not 100% sure but I will specify the branch as main to be sure :)